src/devices/cpu/axc51/axc51.cpp

```diff
@@ -164,10 +189,34 @@ void axc51base_cpu_device::iram_iwrite(offs_t a, uint8_t d) { m_data.write_byte(
 #define IE          SFR_A(ADDR_IE)
 #define IP          SFR_A(ADDR_IP)
 #define B           SFR_A(ADDR_B)
+#define R8          SFR_A(AXC51_ER8)
+
+#define DPL1         SFR_A(AXC51_DPL1)
+#define DPH1         SFR_A(AXC51_DPH1)
+
+
+#define ER00         SFR_A(AXC51_ER00)
+#define ER01         SFR_A(AXC51_ER01)
+
+#define ER10         SFR_A(AXC51_ER10)
+#define ER11         SFR_A(AXC51_ER11)
+
+#define ER20         SFR_A(AXC51_ER20)
+#define ER21         SFR_A(AXC51_ER21)
+
+#define ER30         SFR_A(AXC51_ER30)
+#define ER31         SFR_A(AXC51_ER31)
 
 #define R_REG(r)    m_scratchpad[(r) | (PSW & 0x18)]
 #define DPTR        ((DPH<<8) | DPL)
 
+#define DPTR1        ((DPH1<<8) | DPL1)
+
+#define ER0       ((ER00<<8) | ER01)
+#define ER1       ((ER10<<8) | ER11)
+#define ER2       ((ER20<<8) | ER21)
+#define ER3       ((ER30<<8) | ER31)
```

Comment:
The columns don’t line up here.

Reply:
they should align now, although given the state of flux the files are in this seems premature, would rather concentrate on figuring out why things are going off the rails.